developer-docs-site/docs/community/aptos-style.md

```diff
@@ -176,6 +199,12 @@ A section heading is the title for an individual section within a document page.
     
     - **Preferred**: Execute on your macOS or Linux system
     - **Avoid**: Execute on your macOS/Linux system
+
+## Avoid duplication
+
+We face too many challenges to tackle the same one from scratch again or split our efforts into silos. We must collaborate to make best use of our diverse and growing skillset.
+
+Search and navigate across this site to see if an existing document already serves your purpose and garners an update before starting anew. As with code, [don't repeat yourself](https://www.wikipedia.org/wiki/Don%27t_repeat_yourself).
```

Comment:
We need to reconcile our website with the move book now that they are one source.

Reply:
> We need to reconcile our website with the move book now that they are one source.

Ack. We have removed links to the older book from the main Move pages and are doing the rest of the site as tracked in:
https://github.com/aptos-labs/aptos-core/issues/7364#issuecomment-1485361832